packages/access/src/ownable/ownable.cairo

```diff
@@ -280,7 +290,6 @@ pub mod OwnableComponent {
         fn assert_only_owner(self: @ComponentState<TContractState>) {
```

Comment:
In line 285, we should assert in the `initializer` that the owner is not the zero address, so users won't mistakenly assign that in construction time.

Reply:
Tests should be added for this and the Requirements comment must be updated.